test/unit/ngsi-ld/ngsiService/active-devices-test.js

```diff
@@ -600,7 +600,7 @@ describe('NGSI-LD - Active attributes test', function () {
                 should.exist(error.name);
                 error.code.should.equal(207);
                 error.details.notUpdated.should.equal('someEntities');
-                error.message.should.equal('Error accesing entity data for device: light1 of type: Light');
+                //error.message.should.equal('Error accesing entity data for device: light1 of type: Light');
```

Comment:
Should this line be "recovered" ?

```suggestion
                error.message.should.equal('Error accesing entity data for device: light1 of type: Light');
```

Reply:
Not, current message is more large and include a lot of fields of device.